python/paddle/tensor/math.py

```diff
@@ -5048,6 +5048,105 @@ def digamma_(x, name=None):
         return _C_ops.digamma_(x)
 
 
+def igamma(x, a, name=None):
```

Comment:
use `gammaincc` can better express the meaning of `regularized upper incomplete gamma function` and more better then `igamma`? same issue with `igammac`

Reply:
done